### PR TITLE
issue: 928161 Add man pages

### PR DESCRIPTION
**Pull request by igor-ivanov**
Added man pages for:
- vma.7
- vma_get_api.3
- vma_add_conf_rule.3
- vma_dump_fd_stats.3
- vma_thread_offload.3
- vma_register_recv_callback.3
- vma_recvfrom_zcopy.3
- vma_free_packets.3

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/1936/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/1372/ for details (Mellanox internal link).

File: man/vma_dump_fd_stats.3.in [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+.SH SYNOPSIS
+.B #include <mellanox/vma_extra.h>
+.sp
+.BI "int vma_dump_fd_stats(int " "fd" ", int " "log_level" );
+
+.SH DESCRIPTION
+.B vma_dump_fd_stats()
+Dumps statistics for fd number using log_level log level.
+
```
Comment (liranoz12) [inline man/vma_dump_fd_stats.3.in]: Please consider adding the description of the parameters, for example:
 The argument _fd_ is the file descriptor of the socket, use 0 for all open file descriptors.
 The argument _log_level_ is the dumping level corresponding vlog_levels_t enum (vlogger.h).

**Comment by igor-ivanov** [comment]
>
>1. I notice some copy-paste from User Manual. How would you suggest that we will not repeat the same in both (User manual and man pages). Maybe: give references from User Manual to Man pages.
Of course we can but purpose of man pages to get answer in place.
>2. Where is vmapoll API covered? 
I think we can add one after official releasing these API.
>3. The man pages should be reviewed by technical documentation forum and then go through technical writing editing.
Please join them to the review

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/1953/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/1390/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/1955/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/1522/ for details (Mellanox internal link).

**Comment by igor-ivanov** [comment]
replaced with https://github.com/Mellanox/libvma/pull/896

**Pull request closed without merging**